.circleci/integration-tests/master_dag.py

```diff
@@ -93,6 +93,7 @@ def prepare_dag_dependency(task_info, execution_time):
         {"s3_sensor_dag": "example_s3_sensor"},
         {"redshift_sql_dag": "example_async_redshift_sql"},
         {"redshift_cluster_mgmt_dag": "example_async_redshift_cluster_management"},
+        {"emr_cluster_dag": "example_emr_job_flow_manual_steps"},
```

Comment:
what does "example_emr_job_flow_manual_steps" signify here ? can we please name it as {"emr_job_flow_sensor_dag": "example_emr_job_flow_sensor"}

Reply:
Actually it has to be EMR step sensor {"emr_step_sensor_dag": "example_step_sensor"}